src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
```

Comment:
Change the name of the object, which will not conflict with the `style` attribute
use `stylesObject`, `styleConfig`, `customStyles` for example